docs/semgrep-assistant/overview.md

```diff
@@ -65,26 +65,39 @@ Assistant's suggestions to ignore findings are also surfaced in PR or MR comment
 ![Semgrep Assistant in a Slack notification](/img/semgrep-assistant-slack.png#md-width)
 *Figure*. Semgrep Assistant auto-triage in a Slack notification.
 
-### Autofix Semgrep Code findings
+### Remediation
 
-Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings when it identifies a true positive. Assistant only suggests an autofix if the rule doesn't have a human-written autofix. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, Assistant can recommend its use in the autofix suggestion for the issue in your code.
+Semgrep Assistant can provide remediation advice and autofixes, or suggested fixes, for Semgrep Code findings.
 
-You can set the minimum autofix confidence level required to display autofix suggestions from Semgrep Assistant in Semgrep AppSec Platform's **Settings** page. To receive as many Assistant suggestions as are available, set the minimum to **low confidence**.
+#### Guidance
 
-Autofixes are available in PR and MR comments, so developers can review and verify Semgrep's generated fixes before they're applied.
+With Assistant enabled, every PR or MR comment pushed by Semgrep contains remediation guidance unless the original rule message contains information on how to fix the issue. Assistant's remediation guidance provides step-by-step instructions on how to remediate the finding identified by Semgrep Code.
```

Comment:
> "Unless the original rule message contains information..." 

Does this mean, if there's any content in `message` or if the `message` key is used, there is no remediation guidance?

Reply:
Basically, is it `message`, `fix`, or **both **?

Reply:
No, so we will attempt to generate remediation guidance for all issues. We then do a semantic similarity check to make sure we aren't posting a comment that just echos what was said in the rule message.

Reply:
I'll simplify to saying that we'll generate guidance for all issues; I don't think we need to provide further details on how we choose wording unless you think it necessary?